### PR TITLE
fixed:#2527 Refactored CSS files in src/screens/UserPortal

### LINKED ISSUE #2527: Refactor CSS files in src/screens/UserPortal
# Refactor CSS files in src/screens/UserPortal

To improve the UI/UX of Talawa-Admin we have been working on:
1. making the application more suitable to color blind users
2. streamlining all CSS into a single global file

## Additional information:
1. The foundational work for this was completed in this PR:
    - https://github.com/PalisadoesFoundation/talawa-admin/pull/2466
2. The desired single CSS file is:
    - `src/style/app.module.css`

## The challenge:
The goal is to convert the CSS file in this subdirectory and all the components related to this screen to use this new design pattern. This means that:
1. All CSS files in this subdirectory and all the CSS files in related components must be merged into the global CSS file using pre-existing CSS.
2. All CSS files in this subdirectory and all the CSS files in related components must be deleted after the merge.
3. The remaining relevant files in this subdirectory must only reference the global CSS file.
4. All the CSS files in the related components of this subdirectory must adhere to the same guidelines.

### PR DESCRIPTION
**Pull request by gurramkarthiknetha**
**What kind of change does this PR introduce?**

**Issue Number:**

Fixes #2527 

**Did you add tests for your changes?**
Yes 

**Snapshots/Videos:**

**If relevant, did you update the documentation?**

**Summary**

The goal is to convert the CSS file in this subdirectory and all the components related to this screen to use this new design pattern.

**Does this PR introduce a breaking change?**
NO

**Other information**

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**

Yes 

## Summary by CodeRabbit

- **New Features**
	- Updated styling for various components across the User Portal, including Campaigns, Chat, Donate, Events, Organizations, People, Pledges, Posts, Settings, UserScreen, and Volunteer sections.
	- Introduced new CSS classes and variables in the global style file for enhanced design consistency and responsiveness.

- **Chores**
	- Removed several unused CSS files to streamline the codebase.
	- Modified import paths for styling to ensure consistent application of CSS across components.

### CODE REVIEW

File: src/screens/UserPortal/Volunteer/VolunteerManagement.module.css [hunk 1]
```diff
@@ -1,138 +0,0 @@
...
-}
-
-.noOutline input {
-  outline: none;
-}
-
-.greenregbtn {
-  margin: 1rem 0 0;
-  margin-top: 15px;
-  border: 1px solid #e8e5e5;
```
Comment (palisadoes) [inline src/screens/UserPortal/Volunteer/VolunteerManagement.module.css:123]: It looks like you have truncated this file to zero bytes. Why wasn't it deleted?

File: src/style/app.module.css [hunk 2]
```diff
@@ -308,13 +197,6 @@
   border: 0;
 }
 
-.inputField {
-  margin-top: 10px;
-  margin-bottom: 10px;
```
Comment (palisadoes) [inline src/style/app.module.css:313]: Why was this removed? Won't it have an impact on other layouts?

File: src/style/app.module.css [hunk 3]
```diff
@@ -223,11 +83,36 @@
   margin: 2.5rem 0;
   align-items: center;
   gap: 10px;
+  /* Adjust spacing between items */
+  margin: 2.5rem 0;
 }
 
 .btnsContainer .btnsBlock {
   display: flex;
-  width: max-content;
```
Comment (palisadoes) [inline src/style/app.module.css:230]: Why was this removed? Won't it have an impact on other layouts?

File: src/style/app.module.css [hunk 4]
```diff
@@ -1,160 +1,20 @@
...
+.noOutline input {
+  outline: none;
 }
 
 .noOutline:is(:hover, :focus, :active, :focus-visible, .show) {
   outline: none !important;
 }
 
 .closeButton {
-  margin-bottom: 10px;
```
Comment (palisadoes) [inline src/style/app.module.css:157]: Why was this removed? Won't it have an impact on other layouts?

File: src/style/app.module.css [hunk 5]
```diff
@@ -1,160 +1,20 @@
...
+  --bs-accordion-btn-focus-box-shadow: none;
+  --bs-accordion-border-width: 0px;
 }
 
-.sidebarsticky > input {
-  text-decoration: none;
-  margin-bottom: 50px;
-  border-color: #e8e5e5;
-  width: 80%;
-  border-radius: 7px;
```
Comment (palisadoes) [inline src/style/app.module.css:144]: Why was this removed? Won't it have an impact on other layouts?

File: src/style/app.module.css [hunk 6]
```diff
@@ -1,160 +1,20 @@
...
-  background: #fcfcfc;
-  margin: 10px 20px;
-  padding: 20px 30px;
-  border-radius: 5px;
-  border: 1px solid #e8e8e8;
-  box-shadow: 0 3px 5px #c9c9c9;
-  margin-right: 40px;
-  color: #737373;
-}
-.cards > h2 {
```
Comment (palisadoes) [inline src/style/app.module.css:52]: Why was this removed? Won't it have an impact on other layouts?

File: src/style/app.module.css [hunk 7]
```diff
@@ -349,21 +256,32 @@
...
+  border: 1px solid var(--dropdown-border-color) !important;
 }
 
 .yesButton {
   background-color: var(--search-button-bg);
   border-color: var(--search-button-border);
 }
 
-.searchIcon {
-  color: var(--brown-color);
```
Comment (palisadoes) [inline src/style/app.module.css:360]: Why was this removed? Won't it have an impact on other layouts?

File: src/style/app.module.css [hunk 8]
```diff
@@ -389,10 +307,16 @@
...
+  margin-top: 10px;
+  width: 100%;
+  margin-left: -15px;
+  margin-right: -15px;
+  margin-bottom: 20px;
+}
+
 .rowBackground {
   background-color: var(--row-background);
-  max-height: 120px;
```
Comment (palisadoes) [inline src/style/app.module.css:394]: Why was this removed? Won't it have an impact on other layouts?

File: src/style/app.module.css [hunk 9]
```diff
@@ -401,48 +325,56 @@
...
+}
+
 .errorIcon {
   transform: scale(1.5);
   color: var(--bs-danger);
   margin-bottom: 1rem;
-  /* Add error icon for non-color indication */
-  &::before {
-    content: '⚠️';
-    margin-right: 0.5rem;
```
Comment (palisadoes) [inline src/style/app.module.css:415]: Why was this removed? Won't it have an impact on other layouts?

File: src/style/app.module.css [hunk 10]
```diff
@@ -401,48 +325,56 @@
...
-.subTagsLink:focus i {
+.subTagsLink:hover i {
   visibility: visible;
 }
 
-.manageTagScrollableDiv {
-  scrollbar-width: thin;
-  scrollbar-color: var(--bs-gray-400) var(--bs-white);
-  max-height: calc(100vh - 18rem);
-  overflow: auto;
```
Comment (palisadoes) [inline src/style/app.module.css:439]: Why was this removed? Won't it have an impact on other layouts?

File: src/style/app.module.css [hunk 11]
```diff
@@ -472,11 +404,6 @@ input[type='radio']:checked + label:hover {
   margin-left: 13vw;
 }
 
-.datediv {
-  display: flex;
-  flex-direction: row;
-}
```
Comment (palisadoes) [inline src/style/app.module.css:478]: Why was this removed? Won't it have an impact on other layouts?

File: src/style/app.module.css [hunk 12]
```diff
@@ -542,2503 +469,1119 @@ hr {
   align-items: center;
 }
 
-.pageNotFound {
-  position: relative;
```
Comment (palisadoes) [inline src/style/app.module.css:546]: Why was this removed? Won't it have an impact on other layouts?

File: src/style/app.module.css [hunk 13]
```diff
@@ -542,2503 +469,1119 @@ hr {
...
-  bottom: 20px;
+.toggleBtn:hover {
+  color: var(--bs-primary) !important;
 }
 
-.pageNotFound h3 {
-  font-family: 'Roboto', sans-serif;
-  font-weight: normal;
```
Comment (palisadoes) [inline src/style/app.module.css:552]: Why was this removed? Won't it have an impact on other layouts?

File: src/style/app.module.css [hunk 14]
```diff
@@ -542,2503 +469,1119 @@ hr {
...
-  letter-spacing: 1px;
+.custom_table {
+  border-radius: 20px;
+  background-color: var(--grey-bg-color);
 }
 
-.pageNotFound .brand span {
-  margin-top: 50px;
```
Comment (palisadoes) [inline src/style/app.module.css:557]: Why was this removed? Won't it have an impact on other layouts?

File: src/style/app.module.css [hunk 15]
```diff
@@ -542,2503 +469,1119 @@ hr {
...
-  font-size: 40px;
+.custom_table tbody tr {
+  background-color: var(--dropdown-hover-color);
 }
 
-.pageNotFound .brand h3 {
-  font-weight: 300;
-  margin: 10px 0 0 0;
```
Comment (palisadoes) [inline src/style/app.module.css:563]: Why was this removed? Won't it have an impact on other layouts?

File: src/style/app.module.css [hunk 16]
```diff
@@ -542,2503 +469,1119 @@ hr {
...
+.custom_table tbody tr:hover {
+  background-color: var(--grey-bg-color);
+  box-shadow: 0 0 0 1px rgba(0, 0, 0, 0.1);
 }
 
-.pageNotFound h1.head {
-  font-size: 250px;
-  font-weight: 900;
```
Comment (palisadoes) [inline src/style/app.module.css:568]: Why was this removed? Won't it have an impact on other layouts?

File: src/style/app.module.css [hunk 17]
```diff
@@ -542,2503 +469,1119 @@ hr {
...
-  position: relative;
-  display: inline-block;
-}
-.pageNotFound h1.head span:before,
-.pageNotFound h1.head span:after {
-  position: absolute;
-  top: 50%;
-  width: 50%;
-  height: 1px;
-  background: #fff;
```
Comment (palisadoes) [inline src/style/app.module.css:584]: Why was this removed? Won't it have an impact on other layouts?

File: src/style/app.module.css [hunk 18]
```diff
@@ -542,2503 +469,1119 @@ hr {
...
-}
-.pageNotFound h1.head span:before {
-  left: -55%;
+.custom_table tbody td:focus {
+  outline: 2px solid #000;
+  outline-offset: -2px;
 }
 
-.pageNotFound h1.head span:after {
-  right: -55%;
```
Comment (palisadoes) [inline src/style/app.module.css:592]: Why was this removed? Won't it have an impact on other layouts?

File: src/style/app.module.css [hunk 19]
```diff
@@ -542,2503 +469,1119 @@ hr {
...
-}
+@media (max-width: 520px) {
+  .btnsContainer {
+    margin-bottom: 0;
+  }
 
-.pledgeContainer {
-  margin: 0.6rem 0;
```
Comment (palisadoes) [inline src/style/app.module.css:596]: Why was this removed? Won't it have an impact on other layouts?

File: src/style/app.module.css [hunk 20]
```diff
@@ -542,2503 +469,1119 @@ hr {
...
-}
+  .btnsContainer .btnsBlock {
+    display: block;
+    margin-top: 1rem;
+    margin-right: 0;
+  }
 
-.container {
-  min-height: 100vh;
-}
```
Comment (palisadoes) [inline src/style/app.module.css:601]: Why was this removed? Won't it have an impact on other layouts?

File: src/style/app.module.css [hunk 21]
```diff
@@ -542,2503 +469,1119 @@ hr {
...
+  .btnsContainer .btnsBlock div {
+    flex: 1;
+  }
 
-.pledgeModal {
-  max-width: 80vw;
```
Comment (palisadoes) [inline src/style/app.module.css:604]: Why was this removed? Won't it have an impact on other layouts?

File: src/style/app.module.css [hunk 22]
```diff
@@ -542,2503 +469,1119 @@ hr {
...
+    margin-right: 0.5rem;
+  }
 
-.greenregbtnPledge {
-  margin-top: 15px;
-  border: 1px solid var(--bs-gray-300);
-  box-shadow: 0 2px 2px #e8e5e5;
-  padding: 10px 10px;
-  border-radius: 5px;
-  background-color: var(--bs-primary);
```
Comment (palisadoes) [inline src/style/app.module.css:615]: Why was this removed? Won't it have an impact on other layouts?

File: src/style/app.module.css [hunk 23]
```diff
@@ -542,2503 +469,1119 @@ hr {
...
+    margin-bottom: 1rem;
+    margin-right: 0;
+    width: 100%;
+  }
 }
 
-.btnsContainerPledge {
-  display: flex;
-  gap: 0.8rem;
-  margin: 2.2rem 0 0.8rem 0;
```
Comment (palisadoes) [inline src/style/app.module.css:630]: Why was this removed? Won't it have an impact on other layouts?

File: src/style/app.module.css [hunk 24]
```diff
@@ -542,2503 +469,1119 @@ hr {
...
+  width: 100%;
+  box-sizing: border-box;
+  background: #ffffff;
+  border: 1px solid #0000001f;
+  border-radius: 24px;
 }
 
-.btnsContainerPledge input {
-  outline: 1px solid var(--bs-gray-400);
+.listBox .customTable {
```
Comment (palisadoes) [inline src/style/app.module.css:539]: Why was this removed? Won't it have an impact on other layouts?

File: src/style/app.module.css [hunk 25]
```diff
@@ -542,2503 +469,1119 @@ hr {
...
-  margin: 2.2rem 0 0.8rem 0;
+.listBox {
+  width: 100%;
+  flex: 1;
 }
 
-.btnsContainerPledge .inputPledge {
-  flex: 1;
-  min-width: 18rem;
-  position: relative;
```
Comment (palisadoes) [inline src/style/app.module.css:636]: Why was this removed? Won't it have an impact on other layouts?

File: src/style/app.module.css [hunk 26]
```diff
@@ -542,2503 +469,1119 @@ hr {
...
+  letter-spacing: 0em;
+  text-align: left;
+  color: #000000;
+  border-bottom: 1px solid #dddddd;
+  padding: 1.5rem;
 }
 
-.inputFieldPledge {
-  background-color: white;
-  box-shadow: 0 1px 1px #31bb6b;
```
Comment (palisadoes) [inline src/style/app.module.css:649]: Why was this removed? Won't it have an impact on other layouts?

File: src/style/app.module.css [hunk 27]
```diff
@@ -542,2503 +469,1119 @@ hr {
...
 }
 
-.btnsContainerPledge input {
-  outline: 1px solid var(--bs-gray-400);
+.listBox .customTable {
+  margin-bottom: 0%;
 }
 
-.btnsContainerPledge .inputPledge button {
-  width: 52px;
```
Comment (palisadoes) [inline src/style/app.module.css:644]: Why was this removed? Won't it have an impact on other layouts?

File: src/style/app.module.css [hunk 28]
```diff
@@ -542,2503 +469,1119 @@ hr {
...
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
 }
 
-.dropdownPledge {
-  background-color: white;
-  border: 1px solid var(--bs-primary);
-  position: relative;
```
Comment (palisadoes) [inline src/style/app.module.css:655]: Why was this removed? Won't it have an impact on other layouts?

File: src/style/app.module.css [hunk 29]
```diff
@@ -542,2503 +469,1119 @@ hr {
...
+    margin: 0;
+  }
+
+  .btnsContainer .btnsBlock div button {
+    margin-right: 1.5rem;
+  }
 }
 
-.rowBackgroundPledge {
-  background-color: var(--bs-white);
```
Comment (palisadoes) [inline src/style/app.module.css:661]: Why was this removed? Won't it have an impact on other layouts?

File: src/style/app.module.css [hunk 30]
```diff
@@ -542,2503 +469,1119 @@ hr {
...
+  }
+
+  .listBox .itemCard {
+    width: 100%;
+  }
 }
 
-.TableImagePledge {
-  object-fit: cover;
-  width: calc(var(--table-image-size) / 2) !important;
```
Comment (palisadoes) [inline src/style/app.module.css:667]: Why was this removed? Won't it have an impact on other layouts?

File: src/style/app.module.css [hunk 31]
```diff
@@ -542,2503 +469,1119 @@ hr {
...
+
+  100% {
+    -webkit-transform: rotate(360deg);
+    transform: rotate(360deg);
+  }
 }
 
-.imageContainerPledge {
-  display: flex;
-  align-items: center;
```
Comment (palisadoes) [inline src/style/app.module.css:674]: Why was this removed? Won't it have an impact on other layouts?

File: src/style/app.module.css [hunk 32]
```diff
@@ -542,2503 +469,1119 @@ hr {
...
 
-.pledgerContainer {
+.btnsContainers {
   display: flex;
-  align-items: center;
-  justify-content: center;
-  margin: 0.1rem 0.25rem;
-  gap: 0.25rem;
-  padding: 0.25rem 0.45rem;
-  border-radius: 0.35rem;
```
Comment (palisadoes) [inline src/style/app.module.css:685]: Why was this removed? Won't it have an impact on other layouts?

File: src/screens/UserPortal/Chat/Chat.tsx [hunk 33]
```diff
@@ -105,29 +113,12 @@ export default function chat(): JSX.Element {
     },
   });
 
-  React.useEffect(() => {
+  useEffect(() => {
     if (chatsListData) {
       setChats(chatsListData.chatsByUserId);
```
Comment (palisadoes) [inline src/screens/UserPortal/Chat/Chat.tsx:118]: Why was this file edited for functionality outside of the CSS? We have an open PR to fix chat.

**Review by palisadoes — CHANGES_REQUESTED** [review]
1. There are a lot of unexplained changes to the CSS. Please explain.
2. You also changed the logic in the Chat.tsx file. This is out of scope

File: src/style/app.module.css (hunk 2 above)
Comment (gurramkarthiknetha) [reply src/style/app.module.css:313]: it's already exists on line 112

**Comment by gurramkarthiknetha** [comment]
> 1. There are a lot of unexplained changes to the CSS. Please explain.
> 2. You also changed the logic in the Chat.tsx file. This is out of scope

i will keep new pr with in 5hrs with all required changes 

> 1. There are a lot of unexplained changes to the CSS. Please explain.
> 2. You also changed the logic in the Chat.tsx file. This is out of scope

I will submit a new pull request within 5 hours with all the required changes.

**Comment by gurramkarthiknetha** [comment]
> 1. There are a lot of unexplained changes to the CSS. Please explain.
> 2. You also changed the logic in the Chat.tsx file. This is out of scope

I will submit a new pull request within 5 hours with all the required changes.

**Comment by gurramkarthiknetha** [comment]
image
in the file chart.tsx i getting these error when i try to commit

**Comment by palisadoes** [comment]
Thanks for submitting your PR, but we must close it.

1. We get notified of every event in the queue, and draft PRs like these, and their updates that require no action can be a distraction. We need our reviewers to focus on updates that are ready. 
1.  Don't be discouraged, we are just trying to keep our workloads to a manageable level.

Please resubmit when you are ready.

**Pull request closed without merging**